bigquery_etl/operational_monitoring/templates/histogram_query.sql

```diff
@@ -82,10 +91,14 @@ merged_histograms AS (
   CROSS JOIN
     UNNEST(metrics)
   WHERE branch IN (
+    {% if branches %}
```

Comment:
Likewise, a quick comment on the conditional logic here might help someone looking at this code later